src/panels/lovelace/cards/hui-entity-filter-card.ts

```diff
@@ -12,6 +12,8 @@ class EntityFilterCard extends HTMLElement implements LovelaceCard {
   private _config?: EntityFilterCardConfig;
   private _configEntities?: EntityConfig[];
   private _baseCardConfig?: LovelaceCardConfig;
+  private _hass?: HomeAssistant;
+  private _oldEntities: EntityConfig[] = [];
```

Comment:
The config would never be set when the `entitiesList` will be empty, so let's make it undefined at the start and check for that.
```suggestion
  private _oldEntities?: EntityConfig[];
```